packages/eslint-plugin/tests/rules/return-await.test.ts

```diff
@@ -312,6 +312,27 @@ ruleTester.run('return-await', rule, {
         },
       ],
     },
+    {
+      code: `
+        const fn = (): any => null;
+        async function test() {
+          return await fn();
+        }
+      `,
+      // output matches input because return type any is suggestion only
+      output: `
+        const fn = (): any => null;
+        async function test() {
+          return await fn();
+        }
+      `,
+      errors: [
+        {
+          line: 4,
+          messageId: 'nonPromiseAwait',
```

Comment:
can add suggestion info here: (just found out this was a thing while looking at [this](https://github.com/typescript-eslint/typescript-eslint/pull/2672/files))
```suggestion
          messageId: 'nonPromiseAwait',
          suggestions: [
             {
               messageId: 'nonPromiseAwait',
               output: `
                 const fn = (): any => null;
                 async function test() {
                   return fn();
                 }
               `,
             },
           ],
```

Reply:
Cool - looking at the example, seems that I can drop the outer `output` test too (although I couldn't see a way of not having to break the indentations). Change made.

Reply:
If the internal rule for formatting test cases is giving you grief you can prefix the template string with `noFormat` to disable that rule for that code block.